google/cloud/firestore_v1/aggregation.py

```diff
@@ -127,16 +152,21 @@ def _make_stream(
                 system-specified policy.
             timeout (Optional[float]): The timeout for this request.  Defaults
                 to a system-specified value.
+            explain_options
+                (Optional[:class:`~google.cloud.firestore_v1.query_profile.ExplainOptions`]):
+                Options to enable query profiling for this query. When set,
+                explain_metrics will be available on the returned generator.
 
         Yields:
-            :class:`~google.cloud.firestore_v1.base_aggregation.AggregationResult`:
+            Tuple[Optional[DocumentSnapshot], Optional[ExplainMetrics]]:
```

Comment:
shouldn't `Optional[DocumentSnapshot]` be `List[AggregationResult]`?

Reply:
indeed, updated